tests/Properties.hs

```diff
@@ -2079,7 +2079,7 @@ pl_tests =
     , testProperty "zipWith"          prop_zipWithPL
 --  , testProperty "zipWith"          prop_zipWithCL
```

Comment:
Seems `prop_zipWithCL` is lost long time ago. Could you please remove this line, just while we are at it?

Reply:
Done in 68452b1